nightmare_control/src/nightcontrol.py

```diff
@@ -67,10 +67,10 @@ def hc12_writer():
         
         global buf_x1, buf_x2, buf_y1, buf_y2, buf_curr
         hc.write(HEADER)
-        _lx = round((buf_x1*127)+127) #scale -1..1 values to 0..255 
-        _ly = round((buf_y1*127)+127)
-        _rx = round((buf_x2*127)+127)
-        _ry = round((buf_y2*127)+127)
+        _lx = round(fmap(buf_x1,-1,1,0,255)) #scale -1..1 values to 0..255 
+        _ly = round(fmap(buf_x1,-1,1,0,255))
+        _rx = round(fmap(buf_x1,-1,1,0,255))
+        _ry = round(fmap(buf_x1,-1,1,0,255))
```

Comment:
all lines have `buf_x1` instead of `y1, x2, y2`

Reply:
yeah, I forgot to save before pushing